packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -776,8 +776,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   Rect describeApproximatePaintClip(RenderObject child) => _overflow > 0.0 ? Offset.zero & size : null;
 
   @override
-  String toString() {
-    String header = super.toString();
+  String toShortDescription() {
```

Comment:
I switched all the RenderObject classes to have a toString that returns a full description of the object and properties. If this is a mistake I can switch it back but having the toString methods for RenderObject be inconsistent with other Diagnosticable objects seems odd.